_posts/2020-09-16-creating-a-snap-chat-mini.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "What we Learned Developing Snap Minis (Part 1)"
+date: 2020-09-16 12:00:00
+categories: ["learning"]
+author: arieljuod
+---
+
+Over the last few months, we developed a couple of (Snap Minis)[https://www.ombulabs.com/blog/software-development/snap-minis.html]. Minis are small static web apps that are run inside a webview within the Snapchat native app.
+
+One important part of the development was to make sure our Mini works well across different devices, especially making sure things work the same in both Android's webview (Chrome by default) and iOS's webview (Safari).
+
+In this post I'll talk about a few things to keep in mind when working with the Snap Canvas SDK and in the next one I'll talk about some common and known issues you'll encounter when making your app cross-mobile-browser compatible.
```

Comment:
Is there any public link to the Snap Canvas SDK that we can include?

Reply:
Nope, I understand snapchat wants to handle that privately. There's the Snap Kit SDK, but it's not the same as the Snap Canvas SDK we use for the minis.